packages/openactive-broker-microservice/app.js

```diff
@@ -1357,7 +1274,9 @@ async function processOpportunityItem(item) {
  * @returns {RpdePageProcessor}
  */
 function monitorOrdersPage(orderFeedType, bookingPartnerIdentifier) {
-  return (rpdePage) => {
+  /* Note that the Orders RpdePageProcessor does NOT use validateItemsFn i.e. Orders feed items are not validated
+  LW: I don't know why that is */
```

Comment:
Great point! As Orders are only created in response to bookings, and the Orders feed is specific to the Broker (i.e. the test suite), and as the test suite validates the Order as part of the test, the idea was that there shouldn't be a need to validate what's already in the feed (as they'd have been validated as part of earlier tests).

In controlled mode, where the test data is wiped before each run, the Orders feed is expected to be empty.

So I guess the question becomes should we validate any Orders from previous bookings in Random mode... perhaps it's worth doing this for completeness?


However it could be that Orders ah

Reply:
Could be added as a separate issue if so?

Reply:
Have added as a separate issue (https://github.com/openactive/openactive-test-suite/issues/480) and updated the comment here to explain why it is as it is here